src/ng/directive/form.js

```diff
@@ -268,12 +268,22 @@ FormController.prototype = {
    * @name form.FormController#$setSubmitted
    *
    * @description
-   * Sets the form to its submitted state.
+   * Sets the form to its `$submitted` state. This will also set `$submitted` on all child and
+   * parent forms of the form.
    */
-  $setSubmitted: function() {
-    this.$$animate.addClass(this.$$element, SUBMITTED_CLASS);
-    this.$submitted = true;
-    this.$$parentForm.$setSubmitted();
+  $setSubmitted: function(onlySetOnChildren) {
```

Comment:
The `onlySetOnChildren` param name is misleading as it might imply that we don't set it on `this` :-)
How about something about not setting it on the parent?